contracts/governance/extensions/GovernorCountingSimple.sol

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "../Governor.sol";
+
+/**
+ * @dev Extension of {Governor} for simple, 3 options, vote counting.
+ *
+ * _Available since v4.3._
+ */
+abstract contract GovernorCountingSimple is Governor {
+    string public constant COUNTING_MODULE_NAME = type(GovernorCountingSimple).name;
```

Comment:
I think this should be in `IGovernor`, because it should be defined by every counting module, including `GovernorCompatibilityBravo`.

I don't like the name so much because I think is not so much about the "name" but about the way it operates.

My suggestion: `COUNTING_MODE = "ternary"`.

I'm trying to understand what this value should describe exactly. Definitely the accepted `support` values and their meaning.

What about the quorum calculation? Should this describe whether it counts only For votes (as in Bravo), or all votes, or something else?

Reply:
> What about the quorum calculation? Should this describe whether it counts only For votes (as in Bravo), or all votes, or something else?

I think yes

Reply:
Our decision. Variable name `COUNTING_MODE`. Value is a URL-encoded key value pair.

For the Bravo compatibility module we'll use `COUNTING_MODE = "support=bravo&quorum=bravo"`.

For CountingSimple we'll use `COUNTING_MODE = "support=bravo&quorum=for,abstain"` assuming we change it to be `forVotes + abstainVotes > quorumVotes`.